lib/internal/bootstrap_node.js

```diff
@@ -307,6 +308,24 @@
     return wrappedConsole;
   }
 
+  function setupInspectorCommandLineAPI() {
+    const inspector = process.binding('inspector');
+    const addCommandLineAPIMethod = inspector.addCommandLineAPIMethod;
+    if (!addCommandLineAPIMethod) return;
+
+    const Module = NativeModule.require('module');
+    const path = NativeModule.require('path');
+    const cwd = tryGetCwd(path);
+
+    const consoleAPIModule = new Module('[consoleAPI]');
+    consoleAPIModule.filename = path.join(cwd, consoleAPIModule.id);
+    consoleAPIModule.paths = Module._nodeModulePaths(cwd);
+
+    addCommandLineAPIMethod('require', function require(request) {
+      return consoleAPIModule.require(request);
+    });
```

Comment:
I'd like to focus on landing `require` first since I'm pretty sure that actually *wanting* to access this fake module is a fairly uncommon requirement. I'd rather see it as a potential point of confusion, especially since this "weird" module would magically get into scope as soon as there's no other `module`. I could foresee someone debugging and getting really confused that `module.exports` keeps showing an empty object even though they are *soooo* certain that they assigned properties (until they realize they didn't pause in the right spot).

As for the name - afaik it's not actually shown to the user anywhere. So I'm not sure how much value there is in bike-shedding the perfect name. :)